platform-sdk/swirlds-common/src/testFixtures/java/com/swirlds/common/test/fixtures/Randotron.java

```diff
@@ -70,7 +70,7 @@ private Randotron(final long seed) {
      * @return a random string
      */
     @NonNull
-    public String randomString(final int length) {
+    public String nextString(final int length) {
```

Comment:
all the random javadocs need updating

Reply:
How would you recommend updating them?

Reply:
It's painful to have to go through each, I recognize. But considering we place much value on the correctness of javadocs, I see no other choice than go through the diff and update each one.

Reply:
I don't mind updating them, I'm just not sure how you recommend that I update them. From my perspective they seem suitable descriptions of what each method does, so I'm not sure what changes to make.